src/include/duckdb/function/function.hpp

```diff
@@ -43,6 +43,10 @@ enum class FunctionNullHandling : uint8_t { DEFAULT_NULL_HANDLING = 0, SPECIAL_H
 //!                            but the result might change across queries (e.g. NOW(), CURRENT_TIME)
 //! VOLATILE                -> the result of this function might change per row (e.g. RANDOM())
 enum class FunctionStability : uint8_t { CONSISTENT = 0, VOLATILE = 1, CONSISTENT_WITHIN_QUERY = 2 };
+
+//! Whether or not a function can throw an error or not
+enum class FunctionErrors : uint8_t { CANNOT_ERROR = 0, CAN_THROW_ERROR = 1 };
```

Comment:
I added the same enum in https://github.com/duckdb/duckdb/pull/15152/files#diff-5106c00b793c662b2a549e971963249df4053edda0a690411d9f302b624664a7 (as I needed it for a different reason) - could you unify them?